tensorboard/plugins/mesh/plugin_data.proto

```diff
@@ -26,4 +26,7 @@ message MeshPluginData {
 
   // Shape of underlying data. Cache it here for performance reasons.
   repeated int32 shape = 6;
+
+  // Bitmap of all existing components in addition to this one.
```

Comment:
What does “in addition to this one” mean? In addition to this component?
Which component?

Reply:
Yes, it means "current component"